css/v202209/symbiota/main.css

```diff
@@ -324,6 +324,47 @@ table.styledtable tr.alt td {
   }
 }
 
+.nested-li {
+  margin-left: 1.5em;
+}
+
+.fieldset-like {
```

Comment:
This is a good question. TotalValidatorPro didn't like a <fieldset> element without a parent <form> element. Upon further "research", it seems to make sense _why:_

> If you use the arrow keys to guide your screen reader through the form you will hear the question announced. This is not an efficient way to navigate through a form though, so most screen reader users choose to use the tab key instead. But it is when you use the tab key to navigate through a form that the question is not announced - making it difficult to know what the correct answer might be!

> We fix this by using the `<fieldset>` and `<legend>` elements. These elements work together to tell screen readers that a group of form fields relate to each other, and to provide a label for the group.

Reply:
Since there was no parent `<form>` element herein, it was clear that the styling one sees with a `<fieldset>` is what was desired here, so I just reproduced that styling in a less opinionated element.